vipps-recurring-api.md

```diff
@@ -458,42 +451,43 @@ Example for a subscription every 30th day:
 ```
 
 **Please note:** It is not possible to change intervals. If the user has
-accepted a yearly interval, the agreement can not be changed to a monthly
+accepted a yearly interval, the agreement cannot be changed to a monthly
 agreement. This requires a new agreement and a new consent from the user.
 It _is_ possible to make a monthly agreement and only charge some of the
 months. The general rule: Be as customer friendly and easy to understand
 as possible.
 
 ### Initial charge
 
-**Please note:** Use
-[Campaigns](#campaigns) in combination with initial charge
-if the subscription is cheaper in the beginning than the normal price later.
+**Please note:** If the subscription is cheaper in the beginning than the normal price later, use
+[campaigns](#campaigns) in combination with initial charge.
 If you use `initialcharge` alone for campaigns, users will be confused by how it appears in Vipps,
 as it looks like the full price period starts immediately.
 
 Initial charge will be performed if the `initialcharge` is provided when
-creating an agreement. If there is no initial charge: Don't sent `initialcharge`
+creating an agreement. If there is no initial charge, don't sent `initialcharge`
```

Comment:
Tiny old typo here. Should be `don't send`

Reply:
Oh! I was blind to that one. It took a minute to see it. :')